CMakeLists.txt

```diff
@@ -0,0 +1,211 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# This is the boilerplate for Mbed OS
+
+cmake_minimum_required(VERSION 3.19.0 FATAL_ERROR)
```

Comment:
Could we add an item on our backlog to only require this version if armclang is being used? I don't believe gcc needs this new of a CMake verison.

Reply:
Thanks.
A ticket has been added to the backlog.

Reply:
Could you link me to the ticket please? Thanks.